rhasspynlu/fsticuffs.py

```diff
@@ -171,7 +171,7 @@ def paths_strict(
             olabel = edge_data.get("olabel", "")
             matching_tokens: typing.List[str] = []
 
-            if olabel.startswith("__label__"):
+            if olabel[:9] == "__label__":
```

Comment:
startswith and endswith are pretty slow compared to string slices (need about twice as long)
Since they are used quite a lot in loops I replaced them

Reply:
Thanks! I didn't know this.